shell/platform/embedder/tests/embedder_unittests.cc

```diff
@@ -174,6 +174,25 @@ TEST_F(EmbedderTest, ExecutableNameNotNull) {
   latch.Wait();
 }
 
+TEST_F(EmbedderTest, ImplicitViewNotNull) {
+  auto& context = GetEmbedderContext(EmbedderTestContextType::kSoftwareContext);
+
+  fml::AutoResetWaitableEvent latch;
+  context.AddNativeCallback(
+      "NotifyBoolValue", CREATE_NATIVE_ENTRY([&](Dart_NativeArguments args) {
+        const auto dart_bool = tonic::DartConverter<bool>::FromDart(
+            Dart_GetNativeArgument(args, 0));
+        EXPECT_TRUE(dart_bool);
```

Comment:
This is fine though I'd be tempted to move the bool out, capture by reference then keep the checks at the top-level of the test. Captures are possible since you've used the `CREATE_NATIVE_ENTRY` macro.

Reply:
Fixed!